scripts/drone/steps/lib.star

```diff
@@ -614,7 +614,7 @@ def test_a11y_frontend_step(ver_mode, port = 3001):
       Drone step.
     """
     commands = [
-        "yarn wait-on http://$HOST:$PORT",
+        "npx wait-on@7.0.1 http://$HOST:$PORT",
```

Comment:
Now `wait-on` has been removed from the package.json will `npx` ask to install it? Does this need the `-y` flag to circumvent that?

Reply:
Great spot! I originally tried that, but not only is it not needed, it prevents it from working at all!

This command runs against [node 14](https://github.com/grafana/grafana/pull/77156#discussion_r1376702359), which ships with an npx which doesn't have/need the `-y` flag. Including it causes the command to fail because it's an unrecognised option 😌

Reply:
Ok, sweet. Thanks for confirming. 👍